driven/storage/adapter.go

```diff
@@ -316,158 +258,53 @@ func (sa *Adapter) FindUsers(clientID string, id []string, external bool) ([]mod
 // LoginUser Login a user's and refactor legacy record if need
 func (sa *Adapter) LoginUser(clientID string, current *model.User) error {
 
-	legacyUser, _ := sa.FindUser(clientID, current.ExternalID, true)
-
 	now := time.Now()
 
+	//TODO: Do we still need this migration?
+	//TODO: If so, handle group_memberships
+	//TODO: NEED TO HANDLE ADMINS AND APP USERS SEPARATELY. CURRENTLY THIS IS MIGRATING ADMINS BACK AND FORTH
 	// transaction
```

Comment:
@shurwit this artificial split of one and the same user unique per app is not good and it produces overhead in design and support. This is clear example of the misconception between the UIUC client and the admin client and the shared groups data (as a resource). The app authorisation should be addressed in different than the current way.

Reply:
Nevermind. I had a talk with @petyos and he explained the situation and the design. I will remove the admin login API to stop the cross membership migration